courses/beginner/10.md

```diff
@@ -48,207 +41,183 @@ control 'nginx-modules' do
   desc 'The required NGINX modules should be installed.'
 end
 
-control 'nginx-conf' do
+control 'nginx-conf-file' do
   impact 1.0
-  title 'NGINX configuration'
-  desc 'The NGINX config file should owned by root, be writable only by owner, and not writeable or and readable by others.'
+  title 'NGINX configuration file'
+  desc 'The NGINX config file should exist.'
```

Comment:
Is it supposed to be missing the logic of the describe block?

Reply:
Correct, it needs to simply recap the full as-is example.rb controls from the my_nginx example the students had developed in the previous sections of the class

Reply:
Done